revolut/__init__.py

```diff
@@ -166,6 +166,66 @@ def transaction(self, id):
     # TODO: def transaction_by_request_id(self, request_id):
 
 
+class ClientMerchant(utils._SetEnv):
+    live = False
+    _session = None
+
+    def __init__(self, session):
+        self._set_env(session.access_token)
+        self._session = session
+
+    def _request(self, func, path, data=None):
+        url = urljoin(self.base_url, path)
+        hdr = {"Authorization": "Bearer {}".format(self._session.merchant_token)}
```

Comment:
To trzeba będzie jakoś posprzątać jak potestujemy, zapewne wydzielić `class MerchantSession`

Reply:
Tu jest wgl błąd. Aktualnie jak korzystam z `TemporarySession` to powinno być `self._session._access_token`.
Dobrze myślę?

Reply:
Pewnie tak :)